am-lib/src/test/java/integration/uk/gov/hmcts/reform/amlib/GetResourceDefinitionsWithCreatePermissionIntegrationTest.java

```diff
@@ -36,83 +35,90 @@
     private static DefaultRoleSetupImportService importerService = initService(DefaultRoleSetupImportService.class);
     private ResourceDefinition resourceDefinition;
     private ResourceDefinition otherResource;
+    private String roleName;
+    private String otherRoleName;
 
     @BeforeEach
     void setUp() {
-        importerService.addResourceDefinition(
-            resourceDefinition = createResourceDefinition(serviceName, UUID.randomUUID().toString(), UUID.randomUUID().toString()));
-        importerService.addResourceDefinition(
-            otherResource = createResourceDefinition(serviceName, UUID.randomUUID().toString(), UUID.randomUUID().toString()));
+        roleName = UUID.randomUUID().toString();
+        otherRoleName = UUID.randomUUID().toString();
+        importerService.addRole(roleName, IDAM, PUBLIC, EXPLICIT);
```

Comment:
In other places you used to do `roleName = UUID.randomUUID().toString()`. Be consistent please, especially that below I can see ` importerService.addResourceDefinition(resourceDefinition = ...)

Reply:
👍